src/Core/Entities/User.cs

```diff
@@ -65,6 +65,10 @@ public class User : ITableObject<Guid>, ISubscriber, IStorable, IStorableSubscri
     public bool UnknownDeviceVerificationEnabled { get; set; }
     [MaxLength(7)]
     public string AvatarColor { get; set; }
+    public DateTime? LastPasswordChangeDate { get; set; }
```

Comment:
❓ You seem to be using the time component so shouldn't these be suffixed by `...Time`?

Reply:
The standard across the codebase today is just a `Date` suffix

Reply:
Meh, the inconsistency!